ckanext/switzerland/templates/admin/confirm_remove_showcase_admin.html

```diff
@@ -0,0 +1,22 @@
+{% extends "page.html" %}
+
+{% block subtitle %}{{ _("Confirm Remove") }}{% endblock %}
+
+{% block maintag %}<div class="row" role="main">{% endblock %}
+
+{% block main_content %}
+  <section class="module span6 offset3">
+    <div class="module-content">
+      {% block form %}
+        <p>{{ _('Are you sure you want to remove this user as a Application Admin - {name}?').format(name=c.user_dict.name) }}</p>
```

Comment:
I am just wondering: application is written mostly without Capitalization. Should application admin then be capitalized?